app/views/shared/synchronizer.html.erb

```diff
@@ -9,29 +9,26 @@ end
 
 <% if media_url.present? %>
   <% # Note: "text-align:left;" is required for the widget container, otherwise the seek bar won't render properly %>
-  <div id="synchronizer-widget" style="text-align:left;" data-media-url="<%= media_url %>" data-<%= synch_mode %>-url="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: synch_mode) %>">
-    <div id="working-area" class="row" style="position:relative;">
-      <div id="left-sidebar" class="col-md-6">
-        <div id="media-playback" class="">
-          <div id="ytplayer"></div>
-          <div id="video"><video id="video-player" data-able-player data-include-transcript="false" preload="auto"><track id="video-captions" label="English" kind="captions" srclang="en" src="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: 'captions')%>" /></video></div>
-          <div id="audio"><audio id="audio-player" data-able-player data-include-transcript="false" preload="auto"><track id="audio-captions" label="English" kind="captions" srclang="en" src="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: 'captions')%>" /></audio></div>
+  <div id="synchronizer-widget" style="text-align:left;" data-media-url="<%= media_url %>" data-<%= synch_mode %>-url="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: synch_mode) %>" class="">
+    <div id="working-area" class="d-flex flex-column flex-md-row mx-lg-n2">
+      <div id="left-sidebar" class="w-100 vh-sm-50 h-md-auto px-lg-2" style="">
+        <div id="media-playback" class="" style="">
+          <div id="video"><video id="video-player" class="able-player" data-include-transcript="false" preload="none" data-captions-position="<%= document['dc_type_ssm']&.detect {|x| x =~ /sound/i || x =~ /audio/i } ? 'overlay' : 'bottom' %>"><source src="<%= media_url %>" /><track id="video-captions" label="English" kind="captions" srclang="en" src="<%= bytestream_content_path(catalog_id: params[:id], bytestream_id: 'captions')%>" /></video></div>
         </div><!-- /#media-playback -->
-
-        <div id="interview-metadata" class="inner" data-assigned="true"><%= render partial: 'synchronizer/interview_metadata' %></div><!-- /#interview-metadata -->
       </div><!-- /#left-sidebar -->
 
-      <div id="right-sidebar" class="col-md-6 pull-none">
+      <div id="right-sidebar" class="vh-md-100 min-vh-50 p-0 w-100 px-lg-2" style="max-height:80vh;overflow-y:auto;overflow-x:hidden;">
```

Comment:
```suggestion
      <div id="right-sidebar" class="vh-md-100 min-vh-50 p-0 w-100 px-lg-2 pt-sm-3" style="max-height:80vh;overflow-y:auto;overflow-x:hidden;">
```
May be good to add a little padding to the top of the right sidebar so that its content is closer in alignment to the top of the AblePlayer box.